save-orchestrator/src/main/resources/application.properties

```diff
@@ -1,3 +1,4 @@
+orchestrator.backendUrl=http://backend:5000
```

Comment:
Please put it in `test/application.proeprties` too

Reply:
Added